rust/frontend/src/config.rs

```diff
@@ -112,7 +112,7 @@ fn default_max_payload_size_bytes() -> usize {
 }
 
 fn default_enable_span_indexing() -> bool {
-    false
+    true
```

Comment:
i don't think we want this flag on by default, what happens in local/single-node config?